.env

```diff
@@ -7,3 +7,4 @@ PROVE_BEGIN_BATCH=
 PROVE_END_BATCH=
 L2GETH_API_URL=
 ROLLUPSCAN_API_URL=
+RUST_MIN_STACK=100000000
```

Comment:
What happens without this variable?

Reply:
The prover will dead with "Segmentation Fault"!

Reply:
I've heard that you faced the issue with grpc. But even with json rpc version, too?

Reply:
Yes it is.
The grpc prover didn't work even after adding this environment variable, so I switched to json rpc...